lib/render/RayCaster.cpp

```diff
@@ -385,29 +430,34 @@ bool RayCaster::UserCoordinates::UpdateCoordinates( const RayCasterParams* param
     // Save the data field values and missing values
     size_t numOfVertices = dims[0] * dims[1] * dims[2];
     if( dataField )
+    {
         delete[] dataField;
+        dataField = nullptr;
+    }
     dataField = new float[ numOfVertices ];
-    if( !dataField )
+    if( !dataField )    // Test if allocation successful for 3D buffers.
```

Comment:
The new[] operator will thrown an exception unless you overload it. I.e. if new[] fails this code will terminate.

Reply:
addressed in commit 14a25750